doc/esmvalcore/preprocessor.rst

```diff
@@ -414,6 +414,92 @@ variable items e.g.:
 
 .. _masking of missing values:
 
+
+Using mask files for area/volume/zonal statistics
```

Comment:
Can you move this section after "Missing values masks"?

Reply:
yep on it